documentation/docs/steps/transportRequestUploadRFC.md

```diff
@@ -0,0 +1,105 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+* You have enabled RFC on the ABAP system.
+* You have a user account on the ABAP system where you have assigned the required roles for uploading via RFC.
+* You have created a transport request on the ABAP system, which is the target container of the upload.
+
+## RFC Client
+
+The RFC Client connects to your ABAP system using the [SAP NetWeaver RFC SDK](https://support.sap.com/en/product/connectors/nwrfcsdk.html).
+
+For more information, see [classical SAP connectivity technology RFC](https://help.sap.com/viewer/753088fc00704d0a80e7fbd6803c8adb/1709%20000/en-US/4888068ad9134076e10000000a42189d.html) for detailed information.
+
+To install an RFC library based Connector Client, proceed as follows:
+
+1. Create a Docker image as described in the Git repository [devops-docker-images/node-rfc](https://github.com/rodibrin/devops-docker-images/tree/master/node-rfc).
+1. Import/push your image to your private [Docker Hub registry](https://hub.docker.com/).
```

Comment:
Leave one of the options: either import or push (I am not sure what is frequently used, maybe push?)

Reply:
done, took push